### ISSUE TITLE
Can environment variables be used to configure the server?

### ISSUE THREAD

**Issue by glen-84** [cerbos/cerbos#1700]
I only see `CERBOS_NO_TELEMETRY`. Is it possible to configure the storage and other values using environment variables?

**Comment by alexolivier** [comment]
Yes you can configure configuration value as an environment variable using

> Config values can reference environment variables by enclosing them between `${}`, for example `${HOME}`. Defaults can be set using `${VAR:default}`.


https://docs.cerbos.dev/cerbos/latest/configuration/index.html
